packages/ui/src/app/pages/Bounty/components/BountiesLayout.tsx

```diff
@@ -1,13 +1,59 @@
-import React from 'react'
+import React, { useRef, useState } from 'react'
+import { useTranslation } from 'react-i18next'
 
 import { PageLayout } from '@/app/components/PageLayout'
+import { BountyEmptyFilter, BountyFilters } from '@/bounty/components/BountiesFilters'
+import { BountiesList } from '@/bounty/components/BountiesList'
+import { BountyStatus, QueryExtraFilter, useBounties } from '@/bounty/hooks/useBounties'
+import { BountyOrderByInput } from '@/common/api/queries'
+import { EmptyTab } from '@/common/components/EmptyTab'
+import { MainPanel } from '@/common/components/page/PageContent'
+import { SearchProcess } from '@/common/components/page/SearchProcess'
+import { Pagination } from '@/common/components/Pagination'
+import { useSort } from '@/common/hooks/useSort'
 
 import { BountiesHeader } from './BountiesHeader'
 
 export interface LayoutProps {
-  children: React.ReactNode
+  tilesComponent?: React.ReactNode
+  extraFilter?: QueryExtraFilter<unknown>
+  bountyStatus?: BountyStatus
+  noMember?: boolean
 }
 
-export const BountiesLayout = ({ children }: LayoutProps) => {
-  return <PageLayout header={<BountiesHeader />} main={children} />
+export const BountiesLayout = ({
+  tilesComponent,
+  extraFilter,
+  bountyStatus = 'active',
+  noMember = false,
```

Comment:
There is no need to declare defaults here as undefined will be handled the same as boolean.
```suggestion
  noMember,
```